library/std/src/sys/sync/condvar/no_threads.rs

```diff
@@ -1,25 +1,48 @@
+use crate::cell::RefCell;
 use crate::sys::sync::Mutex;
+use crate::thread::sleep;
 use crate::time::Duration;
 
-pub struct Condvar {}
+pub struct Condvar {
+    value: RefCell<u32>,
```

Comment:
It looks like the value is always accessed via `borrow_mut`, can't this be a `Cell` rather than `RefCell`?

Reply:
Also could this get a descriptive name like `unused_notifications`?